ethstore/cli/tests/cli.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+extern crate tempdir;
+use std::process::Command;
+use tempdir::TempDir;
+use std::fs::File;
+use std::io::Write;
+
+fn run(args: &[&str]) -> String {
+	let output = Command::new("cargo")
+		.args(&["run", "--"])
+		.args(args)
+		.output()
+		.unwrap();
+	assert!(output.status.success());
+	String::from_utf8(output.stdout).unwrap()
+}
+
+#[test]
+fn cli_cmd() {
+	Command::new("cargo")
+		.arg("build")
+		.output()
+		.unwrap();
+
+	let dir = TempDir::new("test-vault").unwrap();
+	println!("tempdir: {}", dir.path().to_str().unwrap());
```

Comment:
stray println

Reply:
@debris looks like I can't push to this branch, can you help rework the patch here?

Reply:
sure!  fixed, rebased and put in a new branch again!